Apps/VisionOS/VisionOS/TutorialViews/HowToTestIdentifyTutorialView.swift

```diff
@@ -0,0 +1,51 @@
+import MarkdownUI
+import SwiftUI
+
+struct HowToTestIdentifyTutorialView: View {
+    static let title = ScreenTitleConfig("How to Test Identify")
+
+    @EnvironmentObject private var viewModel: ViewModel
+
+    var body: some View {
+        ScrollView(.vertical) {
+            VStack(alignment: .leading) {
+                Markdown {
+"""
+You can follow the steps below to verify whether your setup and
+your use of `CustomerIO.shared.identify` works as expected.
+
+Assuming you called identify with an identifier + an email.
```

Comment:
```suggestion
Assuming you called `identify` with an identifier, such as an email.
```

The sentence written as-is feels like you need an identifier *and* an email when the identifier can be an email.